src/backend/db/zapatos/schema.d.ts

```diff
@@ -2165,7 +2165,7 @@ declare module 'zapatos/schema' {
       */
       seen?: boolean | db.Parameter<boolean> | db.DefaultType | db.SQLFragment | db.SQLFragment<any, boolean | db.Parameter<boolean> | db.DefaultType | db.SQLFragment>;
     }
-    export type UniqueIndex = 'task_pkey';
+    export type UniqueIndex = 'task_pkey' | 'uc_task';
```

Comment:
I might suggest a different name than uc_task, for clarity
On that note, what is uc_task meant to be?

Reply:
"uniqueness constraint: task"